conformance/tests/annotations_forward_refs.py

```diff
@@ -88,6 +88,9 @@ def int(self) -> None:  # OK
 
     y: int = 0  # E: Refers to local int, which isn't a legal type expression
 
+    def __init__(self, ClassC: "ClassC") -> None:
```

Comment:
This looks to me like a bug in mypy. This is a deferred evaluation because the annotation is in quotes. The `ClassC` declared in the class body isn't in scope when the evaluation is performed.

Reply:
That said, this test isn't intended to check for this behavior, so we could trivially change it to avoid the mypy bug:

```python
    def __init__(self) -> None:
        self.ClassC = ClassC()
```

Reply:
@yangdanny97, if you're OK with this proposed modification, please push an update.

Reply:
@JelleZijlstra, are you still reviewing the PR, or are you done? If you're good with it, please merge.